openprocurement/bot/identification/databridge/edr_handler.py

```diff
@@ -179,30 +187,31 @@ def get_edr_details(self):
                                               params={"TENDER_ID": tender_data.tender_id}))
             self.until_too_many_requests_event.wait()
             for edr_id in tender_data.edr_ids:
-                response = self.proxyClient.details(edr_id)
+                document_id = generate_req_id()
+                response = self.proxyClient.details(id=edr_id, headers={'X-Client-Request-ID': document_id})
                 if response.status_code == 200:
-                    try:
-                        data = Data(tender_data.tender_id, tender_data.item_id, tender_data.code,
-                                    tender_data.item_name, tender_data.edr_ids,
-                                    response.json().get('data', {}))
-                    except AttributeError as e:
-                        logger.info('Error data type {} {} {}. Message {}'.format(
-                            tender_data.tender_id, tender_data.item_name, tender_data.item_id, e))
+                    file_content = dict(response.json(), **{'meta': {'id': document_id}})
+                    data = Data(tender_data.tender_id, tender_data.item_id, tender_data.code,
+                                tender_data.item_name, tender_data.edr_ids, file_content)
+                    if not isinstance(response.json(), dict):
+                        logger.info('Document_ID: {}. Error data type {} {} {}. Message {}'.format(
+                            document_id, tender_data.tender_id, tender_data.item_name, tender_data.item_id, "Not a dictionary"),
+                            extra=journal_context({"DOCUMENT_ID": document_id}))
                         self.retry_edr_ids_queue.put(Data(tender_data.tender_id, tender_data.item_id, tender_data.code,
                                                           tender_data.item_name, [edr_id], tender_data.file_content))
                     else:
                         self.upload_to_doc_service_queue.put(data)
-                        logger.info('Successfully created file for tender {} {} {}'.format(
-                            tender_data.tender_id, tender_data.item_name, tender_data.item_id),
+                        logger.info('Document_ID: {}. Successfully created file for tender {} {} {}'.format(
```

Comment:
Successfully created file for tender {} {} {} {}.format(tender_id, item_name, item_id, document_id)